gateway-service/src/main/java/org/zowe/apiml/gateway/security/config/SecurityConfiguration.java

```diff
@@ -92,7 +96,10 @@ protected void configure(HttpSecurity http) throws Exception {
             .authorizeRequests()
             .antMatchers(HttpMethod.POST, authConfigurationProperties.getGatewayTicketEndpoint()).authenticated()
             .and()
-            .x509().userDetailsService(x509UserDetailsService())
+            .x509()
+                .x509AuthenticationFilter(apimlX509AuthenticationFilter())
+                .subjectPrincipalRegex("CN=(.*?)(?:,|$)")
```

Comment:
What is the purpose of this regex?

Reply:
This regex is default one, to extract CN from the certificate. I used it there just because I need custom implementation.

Reply:
Thank you